test/functional/feature_backwards_compatibility.py

```diff
@@ -27,9 +27,7 @@
 
 from test_framework.util import (
     adjust_bitcoin_conf_for_pre_17,
-    assert_equal,
-    sync_blocks,
-    sync_mempools,
+    assert_equal
```

Comment:
nit: Please keep the trailing comma to minimize this diff and future diffs, as well as guiding editors with formatting.